Git Formats/Git Blame Ignore Revisions.sublime-syntax

```diff
@@ -21,18 +21,22 @@ variables:
 
 contexts:
   prototype:
-    - include: comments
+    - include: comment-start
+
+  comment-start:
+    - match: '[{{comment_char}}]'
```

Comment:
Injection here becomes `[[#]]`, and I don't think you want that.

Reply:
```suggestion
    - match: \#
```

Reply:
Should I just copy Erlang at this point? It captures all comment characters though:

```yaml
  comments:
    - match: \%+
      scope: punctuation.definition.comment.erlang
      push: comment-body

  comment-body:
    - meta_include_prototype: false
    - meta_scope: comment.line.percentage.erlang
    - match: \n
      pop: 1
```

Reply:
I've pushed relevant changes to maintain a somewhat useful balance between code changes and effort for descriptions and discussions.